reader/views.py

```diff
@@ -1678,15 +1687,9 @@ def index_api(request, title, raw=False):
             if not apikey:
                 return jsonResponse({"error": "Unrecognized API key."})
             return jsonResponse(func(apikey["uid"], Index, j, method="API", raw=raw).contents(raw=raw))
-        else:
-            title = j.get("oldTitle", j.get("title"))
-            try:
-                library.get_index(title)  # getting the index just to tell if it exists
-                # Only allow staff and the person who submitted a text to edit
-                if not request.user.is_staff and not user_started_text(request.user.id, title):
-                   return jsonResponse({"error": "{} is protected from change.<br/><br/>See a mistake?<br/>Email hello@sefaria.org.".format(title)})
-            except BookNameError:
-                pass  # if this is a new text, allow any logged in user to submit
+        elif book and not request.user.is_staff and not user_started_text(request.user.id, title):
```

Comment:
@nsantacruz @edamboritz do you guys know the history of `user_started_text` based on Steves understanding and myself this check seems unnecessary.